conf/configuration.go

```diff
@@ -60,30 +66,48 @@ func Load(configFile string) (*Configuration, error) {
 	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
 	viper.AutomaticEnv()
 
-	if err := viper.ReadInConfig(); err != nil {
-		return nil, err
+	if err := viper.ReadInConfig(); err != nil && !os.IsNotExist(err) {
+		return nil, errors.Wrap(err, "reading configuration from files")
 	}
 
 	config := new(Configuration)
-
 	if err := viper.Unmarshal(config); err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "unmarshaling configuration")
 	}
 
-	return handleNested(config), nil
+	return handleNested(config)
 }
 
 // This is supppper sad. It is b/c the marshal function doesn't work on nested
 // values. The overrides work, but the marshal won't discover them.
 // see: https://github.com/spf13/viper/issues/190
-func handleNested(config *Configuration) *Configuration {
+func handleNested(config *Configuration) (*Configuration, error) {
 	config.JWT.Secret = viper.GetString("jwt.secret")
 
 	config.DB.Driver = viper.GetString("db.driver")
 	config.DB.ConnURL = viper.GetString("db.url")
+	if config.DB.ConnURL == "" && os.Getenv("DATABASE_URL") != "" {
```

Comment:
this should be set-able by setting `NETLIFY_COMMERCE_DATABASE_URL` same as the `PORT`